abaco.go

```diff
@@ -556,12 +540,12 @@ func (as *AbacoSource) Sample() error {
 	as.rowColCodes = make([]RowColCode, 0, as.nchan)
 	ncol := len(keys)
 	for col, g := range as.groupKeysSorted {
-		for row := 0; row<g.nchan; row++ {
-			cnum := row + g.firstchan
+		for row := 0; row<g.Nchan; row++ {
+			cnum := row + g.Firstchan
 			name := fmt.Sprintf("chan%d", cnum)
 			as.chanNames = append(as.chanNames, name)
 			as.chanNumbers = append(as.chanNumbers, cnum)
-			as.rowColCodes = append(as.rowColCodes, rcCode(row, col, g.nchan, ncol))
```

Comment:
rowColCodes always seemed a bit over engineered to me, but I guess they work fine so thats not super relevant. But should they be renamed to "channelGroupCodes" or something?

Reply:
Decided on the phone to leave these as-is for now. Hope to change later, when we have more experience with Abaco and channel groups.